.circleci/config.yml

```diff
@@ -1,21 +1,28 @@
 ---
-version: 2
+version: 2.1
+
+executors:
+  golang:
+    docker:
+    - image: cimg/go:1.16
```

Comment:
Does it mean we do not pick the golang minor release anymore ?

Reply:
This is image used to run this build. Most of which is just calling docker commands. The only reason we need a golang image here is for the bump_version workflow.